### ISSUE TITLE
[Quest] Login Error while app runs in the background after a while (~1 hour)

### ISSUE THREAD

**Issue by f-odhiambo** [opensrp/fhircore#2286]
**Describe the bug**
Error coming from the account manager when the app fails to create the account. You get a notification once the issue is triggered when you try to resume the app 

**To Reproduce**
Steps to reproduce the behaviour:
1. Login 
2. Get access token 
3. Get UUID
4. Get the practitioner details endpoint to get details from HAPI 
5. The successfully authenticate user 
6. Then error occurs 

**Expected behaviour**
Add error handling for this and redirect users to either enter a PIN or authenticate themselves afresh using the username and password 

**Screenshots**
<img src="https://user-images.githubusercontent.com/4540684/234495747-2d58cccb-0a85-4df0-b711-39cf4337a906.png" width="200" height="400" />

**Smartphone (please complete the following information):**
 - Device: Emulator and Actual Device, Pixel 4a
 - Android version: API 33, Android 13

**Related ticket** 
1. #2283

**Comment by f-odhiambo** [comment]
Relatedly here is a reported log 

```
{
    "resourceType": "OperationOutcome",
    "text": {
        "status": "generated",
        "div": "<div xmlns=\"http://www.w3.org/1999/xhtml\"><h1>Operation Outcome</h1><table border=\"0\"><tr><td style=\"font-weight: bold;\">ERROR</td><td>[]</td><td><pre>HAPI-0435: Server method returned resource of type[PractitionerDetails] with no ID specified (IResource#setId(IdDt) must be called)</pre></td>\n\t\t\t</tr>\n\t\t</table>\n\t</div>"
    },
    "issue": [
        {
            "severity": "error",
            "code": "processing",
            "diagnostics": "HAPI-0435: Server method returned resource of type[PractitionerDetails] with no ID specified (IResource#setId(IdDt) must be called)"
        }
    ]
}
```

**Comment by f-odhiambo** [comment]
Related Server side fix https://github.com/opensrp/hapi-fhir-opensrp-extensions/pull/42

**Comment by ndegwamartin** [comment]
@f-odhiambo [the corresponding patch](https://github.com/opensrp/fhircore/pull/2287/commits/114780600003e7fc28710bf398341518e06f465a) was merged to main as part of [PR#2287](https://github.com/opensrp/fhircore/issues/2286). If your QA can confirm it now works we can close this ticket

**Comment by ndegwamartin** [comment]
@f-odhiambo the corresponding [client side patch](https://github.com/opensrp/fhircore/pull/2287/commits/114780600003e7fc28710bf398341518e06f465a) was merged to main as part of [PR#2287](https://github.com/opensrp/fhircore/issues/2286). If your QA can confirm it now works we can close this ticket

**Comment by f-odhiambo** [comment]
Seems to have been resolved , closing for now
